### ISSUE TITLE
House price prediction

### ISSUE THREAD

**Issue by saisaneeth** [saisaneeth/Bharath-intern#1]
# Import necessary libraries
import numpy as np
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.linear_model import LinearRegression
from sklearn.metrics import mean_squared_error
import tensorflow as tf
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

# Load dataset
# Assuming you have a CSV file named 'house_data.csv' with features and prices
house_data = pd.read_csv('house_data.csv')

# Split data into features and target variable
X = house_data.drop('price', axis=1)
y = house_data['price']

# Split data into training and testing sets
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Feature scaling
scaler = StandardScaler()
X_train_scaled = scaler.fit_transform(X_train)
X_test_scaled = scaler.transform(X_test)

# Train a Linear Regression model
lr_model = LinearRegression()
lr_model.fit(X_train_scaled, y_train)

# Evaluate Linear Regression model
lr_predictions = lr_model.predict(X_test_scaled)
lr_rmse = np.sqrt(mean_squared_error(y_test, lr_predictions))
print("Linear Regression RMSE:", lr_rmse)

# Build a neural network using TensorFlow
model = Sequential([
    Dense(64, activation='relu', input_shape=(X_train_scaled.shape[1],)),
    Dense(32, activation='relu'),
    Dense(1)
])

# Compile the model
model.compile(optimizer='adam', loss='mean_squared_error')

# Train the model
model.fit(X_train_scaled, y_train, epochs=50, batch_size=32, verbose=0)

# Evaluate the model
nn_predictions = model.predict(X_test_scaled)
nn_rmse = np.sqrt(mean_squared_error(y_test, nn_predictions))
print("Neural Network RMSE:", nn_rmse)
